### PR TITLE
Update Sample to include MenuFlyout Invocation at Cursor Position

### PR DESCRIPTION
**Pull request by karenbtlai**
## Description

Update sample to include menu flyout invocation by right-clicking / long-touch anywhere on the Grid. 

## Motivation and Context

Used to test internal bug 37171845

### CODE REVIEW

File: dev/CommonStyles/TestUI/MenuFlyoutPage.xaml [hunk 1]
```diff
@@ -9,7 +9,27 @@
...
+            </MenuFlyoutSubItem>
+            <MenuFlyoutSeparator/>
+            <MenuFlyoutItem Text="MenuFlyoutItem"/>
+        </MenuFlyout>
+    </local:TestPage.Resources>
+    
+    <Grid Margin="12" ContextRequested="Grid_ContextRequested">
+        <Grid.Resources>
+            
+        </Grid.Resources>
```
Comment (chingucoding) [inline dev/CommonStyles/TestUI/MenuFlyoutPage.xaml:32]: Do we need this empty resource dictionary?

File: dev/CommonStyles/TestUI/MenuFlyoutPage.xaml.cs [hunk 2]
```diff
@@ -12,15 +13,33 @@ namespace MUXControlsTestApp
...
             TestMenuFlyoutItemHeightTextBlock.Text = $"{TestMenuFlyoutItem.ActualHeight}";
             TestMenuFlyoutItemWidthTextBlock.Text = $"{TestMenuFlyoutItem.ActualWidth}";
         }
+
+        private void Grid_ContextRequested(UIElement sender, ContextRequestedEventArgs args)
+        {
+            Point point;
+            var requestedElement = sender as FrameworkElement;
+
+            if (args.TryGetPosition(requestedElement, out point))
```
Comment (chingucoding) [inline dev/CommonStyles/TestUI/MenuFlyoutPage.xaml.cs:35]: nit: You could also use the following and remove line 32:

```suggestion
            if (args.TryGetPosition(requestedElement, out Point point))
```

**Review by chingucoding — APPROVED** [review]

**Review by ojhad — APPROVED** [review]

**Pull request merged**